WooCommerce/Classes/ViewRelated/Themes/ThemesCarouselView.swift

```diff
@@ -0,0 +1,130 @@
+import SwiftUI
+import Kingfisher
+
+
+struct ThemesCarouselView: View {
+    /// Tuple array containing theme name and theme Url string pair.
+    let themesInfo: [(name: String, url: String)] = [
+        (name: "Google", url: "google.com"),
+        (name: "Facebook", url: "facebook.com"),
+        (name: "WordPress", url: "wordpress.com"),
+        (name: "Microsoft", url: "microsoft.com"),
+        (name: "Amazon", url: "amazon.com"),
+        (name: "Apple", url: "apple.com")
+    ]
+
+    /// The title of the message at the end of the carousel.
+    let lastMessageHeading: String
+
+    /// The content of the message at the end of the carousel.
+    let lastMessageContent: String
+
+
+    /// Scale of the view based on accessibility changes
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    var body: some View {
+        ScrollView(.horizontal, showsIndicators: false) {
+            HStack() {
+
+                // Demo for correct state
+                ForEach(themesInfo, id: \.name) { theme in
+                    if let themeUrl = getThemeUrl(themeUrl: theme.url) {
+                        ThemeImage(url: themeUrl)
+                    }
+                }
+
+                // Demo for error state
+                ThemeName(name: themesInfo[0].name)
+
+                // Demo for last message content
+                VStack {
+                    Text(lastMessageHeading)
+                        .bold()
+                        .bodyStyle()
```

Comment:
Nit: the text in the design seems to have secondary color.

<img width="245" alt="image" src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/cc445b20-a724-4c00-befc-98745f4bb831">

Reply:
Handled in https://github.com/woocommerce/woocommerce-ios/pull/11336/commits/b957e7b29b97691e65b15f9c5b98ba0ce42e9022